README.md

```diff
@@ -1,38 +1,81 @@
+
+Newdle is part of the [MALT project](https://malt.web.cern.ch).
+
+## Why another tool?
+
+
+It is true that there is an abundance of similar tools like e.g. [Nuages](https://nuages.domainepublic.net) or [croodle](https://github.com/jelhan/croodle) which are just two of many others available out there. Despite that, we still believe that newdle will provide users with more features and will offer much better user experience. We are putting huge efforts in order to be sure that newdle will meet the following requirements:
+
+- clean and user-friendly UI;
+- integration with [Indico](https://getindico.io);
+- integration with enterprise calendars;
+- data privacy.
 
 ## Development
 
-Run `make` to setup your virtualenv and install all dependencies.
-If Python 3.7 is not in your PATH as `python3.7`, you can use e.g. `PYTHON=python3 make`
-instead (or even point to the full path where your Python 3.7 interpreter is located).
+We chose Python 3.7 as the backend language, so make sure you have it installed. To prepare the development environment it is enough to run `make` which takes care of installing all required dependencies inside a new virtualenv. Typically that will be the `.venv` directory unless you override the environment variable `VENV` *e.g.* `VENV=.virtualenv make`.
+
+Make sure you have the `python3.7` binary in your PATH. You can also use the `PYTHON` environment variable to override the location of the
+`python` binary. *e.g.:*
+```bash
+$ PYTHON=/usr/bin/python3.7 make
+```
+
+## Database schema
+
+Before running the alembic migrations make sure you  have created a database called `newdle` (or adjust the config file). Having done that, run `flask db upgrade` to upgrade the schema.
+
+**:warning: Schema changes :warning:**
+
+At the initial stages of the development it might be pretty common to change the DB schema which also incurs modifications to the SQLAlchemy models. To facilitate the process of keeping actual schema and models in sync we provide `make newdb` command. The purpose of it is to update the initial alembic revision (according to the models declared) and recreate your local database. This is a destructive operation! This command will be removed as soon as there are more alembic revisions.
+
+
+## Running the development server
+
+To run the dev servers, use `make flask-server` and `make react-server` (in separate terminals). You can use the `FLASK_HOST`, `FLASK_PORT` and `REACT_PORT` environment variables to override where the dev servers will listen (make sure to set it for both dev servers, since the react server needs to know where the Flask app is running).
```

Comment:
```suggestion
To run the dev servers, use `make flask-server` and `make react-server` (in separate terminals). You can use the `FLASK_HOST`, `FLASK_PORT` and `REACT_PORT` environment variables to override where the dev servers will listen (make sure to set it for both dev servers, since the React server needs to know where the Flask app is running).
```